packages/react-components/src/components/SendBox.tsx

```diff
@@ -242,9 +248,26 @@ export const SendBox = (props: SendBoxProps): JSX.Element => {
     // Message can be empty if there is a valid attachment upload
```

Comment:
Please use `useCallback` here for `sendMessageOnClick` so it won't be cached between re-renders